apps/snitch_core/lib/core/data/schema/review/review.ex

```diff
@@ -35,7 +35,7 @@ defmodule Snitch.Data.Schema.Review do
   @optional_params ~w(title locale approved)a
 
   @create_params @required_params ++ @optional_params
-  @update_params ~w(description)a ++ @optional_params
+  @update_params ~w(description user_id)a ++ @optional_params
```

Comment:
please remove `user_id` user_id should not be updated for a revenue.

Reply:
@arjun289 sir But if we don't include user_id in updated params then how will we able to check foreign key constraint property.

Reply:
It needs to be checked only during `create` action no need to check it during update.